README.md

```diff
@@ -0,0 +1,216 @@
+# xylem
+
+Library to interact with CarbonCulture's datastore
+
+
+## Installation
+
+Installation using pip::
+
+    pip install https://github.com/CarbonCulture/xylem/archive/v0.4.11.zip
```

Comment:
Yeah, I think we just have to keep this updated in line with CHANGES and the version numbers